src/python_testing/matter_testing_infrastructure/data_model_xmls.gni

```diff
@@ -0,0 +1,564 @@
+# Copyright (c) 2024 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//build_overrides/chip.gni")
+
+data_model_XMLS = [
```

Comment:
Is there a way to do this without hardcoding the individual files? This file set will change, particularly for master.

Reply:
Yes this file `data_model_xmls.gni` is generated via python code which is in the same directory and it got merged earlier to this PR `generate_data_model_xmls_gni.py`. You can run this script and that will output the data model gni file in this same directory.

Reply:
We can later make it list files at build time instead of using a gni. Need to figure out what the best way is here as we do not update these often anyway, so we could also make the "regen" part of the latest XML pull script.